Fix map height on mobile devices (#1459)
hpi-sam/digital-fuesim-manv: CHANGELOG.md

- `DFM_UPLOAD_LIMIT` values are now correctly applied again, after a bug in version 0.11.0
- The vehicle status display now uses the correct visible status of a patient. Thus, after a patient that has been triaged by personnel on the map, the color determined in this triage is used.
- Permissions for multiple actions around simulated regions that were mistakenly allowed for participants have been changed to trainers only.

> - The map height on mobile devices was larger than the screen.

Reviewer: We should include the actual change
```suggestion
- The exercise map now fits on mobile device screens
```
